src/squeeze_xen.ml

```diff
@@ -370,9 +375,10 @@ module Domain = struct
 end
 
 (** Mark the domain as squeezed declared stuck *)
-let declare_domain_stuck ~xc domid =
+let declare_domain_stuck ~xc domid value =
```

Comment:
Not really - Just added the flexibility to add it in future.
I will make changes to rename the function